crates/modelardb_server/src/storage/uncompressed_data_manager.rs

```diff
@@ -585,21 +588,21 @@ mod tests {
 
     /// Create a record batch with data that resembles uncompressed data with a single tag and two
     /// field columns. The returned data has `row_count` rows, with a different tag for each row.
-    /// Also create model table metadata for a model table that matches the created data.
+    /// Also create model table metadata and expr for a model table that matches the created data.
```

Comment:
[`expr` is just a shorthand for `expression`](https://www.abbreviations.com/abbreviation/expression). I have removed `expr` as the original expression is now part of `ModelTableMetadat` instead of being a separately returned value.